tasks/main.yml

```diff
@@ -11,8 +11,8 @@
   when: dell_dsu_manage_multipath_config
 
 - name: Install Dell DSU repo
-  environment: "{{ proxy_env|default({}) }}"
-  shell: "curl -O https://linux.dell.com/repo/hardware/dsu/bootstrap.cgi && yes {{ 'y' if dell_dsu_rollout_gpg_keys_from_dell_script else 'n' }} | bash bootstrap.cgi"
+  environment: "{{ proxy_env | combine( { 'IMPORT_GPG_CONFIRMATION': 'yes' } ) | default({}) }}"
```

Comment:
This works, but I don't know why as I feel the script should overwrite it.

Reply:
Changed to replace the specific variable default to yes (so it will import GPG keys)